### DISCUSSION TITLE
Suggestion: Evaluate stitches

### DISCUSSION THREAD

**Discussion by kvnwolf** [mantinedev/mantine#33] [Feedback]
Hello, awesome library, can't wait to implement it in my next project!

For the maintainers: have you thought of using [stitches](https://stitches.dev) to create the Design System? It has a lot of benefits that you could look at on their homepage.

For instance, components could return a `css` prop, that allows using values from the theme rather than from a hook (which could cause performance issues). Also, all theme tokens are saved as CSS variables, which makes it more performant and delegates styling stuff to CSS rather than JS.

But overall, awesome library, I completely love the Notifications System. **Great work!**

**Comment by rtivital** [comment]
Hi, thanks for feedback, but for stitches – you like five month late. We already have large code base with react-jss, we cannot allocate so much resources to overwrite all components, theming and dynamic values to use another styling option.
